### PR TITLE
chore: Add changesets

### PR DESCRIPTION
**Pull request by jjtang1985**
We need changeset files (release notes) for the GA.
Here is the feature list from high level.

### CODE REVIEW

File: .changeset/kudos-to-the-great-team.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+"@sap-ai-sdk/orchestration": major
+---
+
+[New Functionality] Introduce a new Orchestration client for consuming the following features of the Orchestration service:
```
Comment (marikaner) [inline .changeset/kudos-to-the-great-team.md:5]:
```suggestion
[New Functionality] Introduce orchestration client for consuming the following features of the orchestration service:
```

File: .changeset/kudos-to-the-great-team.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+- Harmonized LLM access via orchestration
+- Use client side prompt templates
```
Comment (marikaner) [inline .changeset/kudos-to-the-great-team.md:7]:
```suggestion
- client side prompt templates
```

File: .changeset/kudos-to-the-great-team.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+---
+"@sap-ai-sdk/orchestration": major
+---
+
+[New Functionality] Introduce a new Orchestration client for consuming the following features of the Orchestration service:
+- Harmonized LLM access via orchestration
```
Comment (marikaner) [inline .changeset/kudos-to-the-great-team.md:6]:
```suggestion
- harmonized LLM access via orchestration
```

File: .changeset/kudos-to-the-great-team.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+- Use client side prompt templates
+- Configure content filtering
```
Comment (marikaner) [inline .changeset/kudos-to-the-great-team.md:8]:
```suggestion
- content filtering
```

File: .changeset/sap-cloud-sdk-for-ai.md [hunk 5]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ai-sdk/ai-api': major
+---
+[New Functionality] Provide an Http Wrapper for consuming the [AI Core Rest APIs](https://api.sap.com/api/AI_CORE_API/overview).
+Thanks to the SAP Cloud SDK OpenAPI Generator, which is used for generating the client.
```
Comment (marikaner) [inline .changeset/sap-cloud-sdk-for-ai.md:5]:
```suggestion
[New Functionality] Provide an HTTP wrapper to consume the [AI Core Rest APIs](https://api.sap.com/api/AI_CORE_API/overview). It is powered by the SAP Cloud SDK OpenAPI generator.
```

File: .changeset/ga-release.md [hunk 6]
```diff
@@ -0,0 +1,4 @@
+---
+'@sap-ai-sdk/foundation-models': major
+---
+[New Functionality] [New Functionality] Offer an OpenAI client to consume Azure OpenAI models for chat completion and embeddings via generative AI hub of SAP AI Core.
```
Comment (marikaner) [inline .changeset/ga-release.md:4]:
```suggestion
[New Functionality] Offer an OpenAI client to consume Azure OpenAI models for chat completion and embeddings via generative AI hub of SAP AI Core. 
```

File: .changeset/ga-release.md [hunk 7]
```diff
@@ -0,0 +1,4 @@
+---
+'@sap-ai-sdk/foundation-models': major
+---
+[New Functionality] Offer an OpenAI client to consume Azure OpenAI models for chat completion and embeddings via generative AI hub of SAP AI Core.
```
Comment (marikaner) [inline .changeset/ga-release.md:4]:
```suggestion

[New Functionality] Offer an OpenAI client to consume Azure OpenAI models for chat completion and embeddings via generative AI hub of SAP AI Core. 
```

File: .changeset/gen-ai-hub.md [hunk 8]
```diff
@@ -0,0 +1,4 @@
+---
+'@sap-ai-sdk/langchain': major
+---
+[New Functionality] Support using the [LangChain](https://js.langchain.com/v0.2/docs/tutorials/) framework for consuming Azure OpenAI models from generative AI hub of SAP AI Core.
```
Comment (marikaner) [inline .changeset/gen-ai-hub.md:4]:
```suggestion

[New Functionality] Support using the [LangChain](https://js.langchain.com/v0.2/docs/tutorials/) framework for consuming Azure OpenAI models from generative AI hub of SAP AI Core.
```

File: .changeset/sap-cloud-sdk-for-ai.md [hunk 9]
```diff
@@ -0,0 +1,4 @@
+---
+'@sap-ai-sdk/ai-api': major
+---
+[New Functionality] Provide an HTTP wrapper to consume the [AI Core Rest APIs](https://api.sap.com/api/AI_CORE_API/overview). It is powered by the SAP Cloud SDK OpenAPI generator.
```
Comment (marikaner) [inline .changeset/sap-cloud-sdk-for-ai.md:4]:
```suggestion

[New Functionality] Provide an HTTP wrapper to consume the [AI Core Rest APIs](https://api.sap.com/api/AI_CORE_API/overview). It is powered by the SAP Cloud SDK OpenAPI generator.
```

**Review by marikaner — APPROVED** [review]
I was so free to make changes.

File: .changeset/gen-ai-hub.md [hunk 10]
```diff
@@ -0,0 +1,5 @@
+---
+'@sap-ai-sdk/langchain': major
+---
+
+[New Functionality] Support using the [LangChain](https://js.langchain.com/v0.2/docs/tutorials/) framework for consuming Azure OpenAI models from generative AI hub of SAP AI Core.
```
Comment (ZhongpinWang) [inline .changeset/gen-ai-hub.md:5]:
```suggestion
[New Functionality] Support using the [LangChain](https://js.langchain.com/docs/tutorials/) framework for consuming Azure OpenAI models from generative AI hub of SAP AI Core.
```
0.2 is no longer maintained.

**Pull request merged**